integration-tests/contract-api-scenario.spec.ts

```diff
@@ -74,6 +74,18 @@ CONFIGS.forEach(({ lib, rpc, setup, knownBaker, createAddress, protocol }) => {
       done();
     })
 
+    it('Return undefined when BigMap key is not found', async () => {
+      const myContract = await Tezos.contract.at(rpc === "https://api.tez.ie/rpc/carthagenet" ? "KT1MLLuyxgxWzW8iRMrNVeamerasADFqiKPW" : "KT1T2KjQdqeNzeaSGm9MfzfgMN8rWC94BrTP");
+      const contractStorage: any = await myContract.storage();
+      let value;
+      if (rpc === "https://api.tez.ie/rpc/carthagenet") {
+        value = await contractStorage[0].get("tz1bwsEWCwSEXdRvnJxvegQZKeX5dj6oKEyz")
```

Comment:
It would probably be easier to use a contract with a similar implementation to avoid different test logic per network.

I deployed a contract with similar storage as the one that is used for babylonnet on carthagenet: https://better-call.dev/carthage/KT1HqWsXrGbHWc9muqkApqWu64WsxCU3FoRf

Here is the link to ligo ide: https://ide.ligolang.org/p/BNv2SLMXnRvKGrB2Q2gjIg this is what I used to deploy this new contract on carthagenet

Reply:
@carte7000 do you think we should make a note of this process of a) how to find the necessary smart contract code (in Michelson/Ligo or w/e was used to launch it) and b) quickest way to launch the same contract on a different network?